__tests__/__fixtures__/docs/existing-docs/subdir/another-doc.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: This is another document title
 category:
-  uri: category-slug
+  uri: /branches/stable/categories/guides/category-slug
```

Comment:
per [my feedback here](https://github.com/readmeio/rdme/pull/1261/files#r2151151087), this brings back the full path so we can test out [the logic we have in place](https://github.com/readmeio/rdme/blob/c88fe69d4b9e54f98333ffde5c7351b8b500f041/src/lib/syncPagePath.ts#L100-L109) for normalizing category URIs. i also created a corresponding directory so we can test this for reference pages as well.